### ISSUE TITLE
Error when running synthesize on blank project clone

### ISSUE THREAD

**Issue by HackXIt** [inclusion-international/speech-jokey#8]
> @Speech jokey team: 
> I'm currently testing the new version of Speech Jokey and I keep getting an error when trying to convert the text into a *.wav file. Do you know what might be causing this?
```
[INFO   ] [MainScreen          ] Selected voice: Rachel
<api.elevenlabsapi.elevenlabsapi.ElevenLabsAPI object at 0x0000024102834D50>
hello guys, this is me.. Natascha. I'm testing the new version of speech jokey... It works great, although my Toby is super slow. This is ridiculous.. But the usability of the software is absolutely fine.
[DEBUG  ] [Starting new HTTPS connection (1)] api.elevenlabs.io:443
[DEBUG  ] [https          ]//api.elevenlabs.io:443 "GET /v1/voices HTTP/1.1" 200 47397
[DEBUG  ] [Starting new HTTPS connection (1)] api.elevenlabs.io:443
[DEBUG  ] [https          ]//api.elevenlabs.io:443 "POST /v1/text-to-speech/pMsXgVXv3BLzUgSXRplE?output_format=mp3_44100_128 HTTP/1.1" 200 335621
[ERROR  ] [MainScreen          ] Error during synthesis: [Errno 2] No such file or directory: 'tmp/output_file.wav' 
```

**Comment by HackXIt** [comment]
Potential future error with OS fixed in: 7fe8c99faaa7dbc1ba8875f05d675ef5f106dc02

Described error from User fixed in: a31c7f4a80cc13c0f357adfeb7ffde9df87eac4d

**Comment by HackXIt** [comment]
If you want to bring the fixing commits over to the other branch where user is testing things, you'll need to do this:
`git cherry-pick <COMMIT-HASH>`
in this specific case:
`git cherry-pick a31c7f4a80cc13c0f357adfeb7ffde9df87eac4d`

Obviously you'll need to have pulled that commit BEFOREHAND, so if you don't have it, please do
`git pull --all`
or
`git checkout develop_ui_fixes && git pull`
and then
`git checkout develop_breaks && git cherry-pick a31c7f4a80cc13c0f357adfeb7ffde9df87eac4d`

**Comment by IsabellaReiher** [comment]
Thank you! I added the commits to the branch with the cherry-pick commands

**Comment by deinhofer** [comment]
fixed by #9
